http2/src/main/java/com/blazemeter/jmeter/http2/visualizers/SimpleDataWriter.java

```diff
@@ -1,92 +0,0 @@
-package com.blazemeter.jmeter.http2.visualizers;
-
-import java.awt.BorderLayout;
-
-import org.apache.jmeter.reporters.ResultCollector;
-import org.apache.jmeter.samplers.SampleResult;
-import org.apache.jmeter.threads.ListenerNotifier;
-import org.apache.jmeter.visualizers.gui.AbstractVisualizer;
-
-
-
-/**
- * This listener can record results to a file but not to the UI. It is meant to
- * provide an efficient means of recording data by eliminating GUI overhead.	
- */
-public class SimpleDataWriter extends AbstractVisualizer{
```

Comment:
Why you removed this class? 
What will happen if I'll open jmx file, that contains this listener?

Reply:
The class no longer makes sense since these changes allow jmeter built-in SimpleDataWriter to accomplish the same goal. If some JMX contains this custom SimpleDataWriter, it will fail with the new version. We should consider setting a proper version (2.0) for communicating the non backwards compatible change, and add proper notice in release notes.

Reply:
If we'll remove some classes we should set version 2.0
But I think, that we can save empty class 
```
public class SimpleDataWriter extends AbstractVisualizer {
}
```

Reply:
I recovered this listener with a deprecated label

Reply:
There is special method to override "getMenuCategories", which you can use to remove component from JMeter menus, while still having old test plans to load it.

Reply:
@sebastianlorenzo88 please look at the comment above

Reply:
done